src/components/LoginPortalToggle/LoginPortalToggle.module.css

```diff
@@ -22,13 +22,13 @@
 }
 
 .activeLink {
-  color: var(--bs-white);
+  color: white;
   border: 1px solid var(--bs-primary);
-  background-color: var(--active-button-bg);
+  background-color: #1e4e8c;
 }
 
 .activeLink:hover {
   color: var(--bs-white);
-  background-color: var(--active-hover);
+  background-color: #1e4e8c;
```

Comment:
Convert to a variable. There should be some preexisting ones, if not please create it.